src/mmw/js/src/app.js

```diff
@@ -13,11 +14,19 @@ var App = new Marionette.Application({
         this.restApi = new RestAPI();
         this.map = new models.MapModel();
 
+        // If in embed mode we are be default in activity mode.
+        var activityMode = settings.get('itsi_embed');
+        settings.set('activityMode', activityMode);
```

Comment:
Trying to stick with camelCase throughout the application where possible we use snake_case where we are getting attributes directly from the API so we don't have to do nasty conversion of key formats. I think that was decided to be the lesser evil though I think that is the exception rather than the rule.